pkg/operator/ceph/cluster/mgr/mgr.go

```diff
@@ -55,6 +55,7 @@ const (
 	standbyMgrStatus       = "standby"
 	monitoringPath         = "/etc/ceph-monitoring/"
 	serviceMonitorFile     = "service-monitor.yaml"
+	serviceMonitorPort     = "http-metrics"
```

Comment:
How do we configure this by CR, can please tell me which specific CRD and in which component